src/MongoMemoryServer.ts

```diff
@@ -97,25 +97,29 @@ export default class MongoMemoryServer {
   }
 
   async _startUpInstance(): Promise<MongoInstanceDataT> {
-    const data = {};
-    let tmpDir;
+    const data: any = {};
+    let tmpDir: SynchrounousResult;
 
     const instOpts = this.opts.instance;
-    data.port = await getport({ port: instOpts.port });
-    this.debug = Debug(`Mongo[${data.port}]`);
-    this.debug.enabled = !!this.opts.debug;
-    data.dbName = generateDbName(instOpts.dbName);
+    data.port = await getPort({ port: (instOpts && instOpts.port) || undefined });
+    /*
+      this.debug = Debug(`Mongo[${data.port}]`); // TODO: Why do we dont just use this.debug here ?
+      this.debug.enabled = !!this.opts.debug; // Useful ?
+    */
+    this.debug(`Mongo[${data.port}]`)
```

Comment:
@nodkz Why do we need this Debug module in the commented code and not just the `this.debug` function to log the port like I did ?

Reply:
If we have several instances eg for ReplicaSet, or in one process created manually several servers.
We should have correct prefixes for every mongod instance in debug.

So `Debug(`Mongo[${data.port}]`);` helps to create prefixied output for instance logs.

To fix this, we need to improve `debug` method in the `MongoInstance` class where will be added `Mongo[${data.port}]` to every log message.